### PR TITLE
[auth-docs] - Add Grafana Cloud Limitations

### PR DESCRIPTION
**Pull request by SeamusGrafana**
Add additional comments to specify what is not available on Grafana Cloud.

**What is this feature?**

Update to Docs, to make clearer what auth methods can't be enabled on Grafana Cloud.

**Why do we need this feature?**

To make it clearer for customer what methods and features are available in OSS/Enterprise (On-Prem) vs Grafana Cloud.

Internal additional context can also be provided if needed

**Who is this feature for?**

Readers of our Documentation, Grafana Cloud Users, Grafana Support.

**Which issue(s) does this PR fix?**:

None

**Special notes for your reviewer:**

Please check that:
- [X] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [X] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-security/configure-authentication/_index.md [hunk 1]
```diff
@@ -33,9 +33,13 @@ The following table shows all supported authentication providers and the feature
 
 ## Grafana Auth
 
-Grafana of course has a built in user authentication system with password authentication enabled by default. You can
-disable authentication by enabling anonymous access. You can also hide the login form and only allow login through an auth
-provider (listed above). There are also options for allowing self sign up.
+Grafana OSS and Grafana Enterprise, of course has a built in user authentication system with password authentication enabled
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/_index.md:36]:
```suggestion
Grafana OSS and Grafana Enterprise, has a built-in user authentication system with password authentication enabled 
```

File: docs/sources/setup-grafana/configure-security/configure-authentication/_index.md [hunk 2]
```diff
@@ -33,9 +33,13 @@ The following table shows all supported authentication providers and the feature
 
 ## Grafana Auth
 
-Grafana of course has a built in user authentication system with password authentication enabled by default. You can
-disable authentication by enabling anonymous access. You can also hide the login form and only allow login through an auth
-provider (listed above). There are also options for allowing self sign up.
+Grafana OSS and Grafana Enterprise, of course has a built in user authentication system with password authentication enabled 
+by default. For Grafana Cloud, the default authentication system uses your `grafana.com` account for authentication.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/_index.md:37]:
```suggestion
by default. For Grafana Cloud, the default authentication system uses your `grafana.com` account for authentication.  
```

File: docs/sources/setup-grafana/configure-security/configure-authentication/_index.md [hunk 3]
```diff
@@ -95,6 +99,8 @@ org_role = Viewer
 
 If you change your organization name in the Grafana UI, this setting needs to be updated to match the new name.
 
+> Please Note: Anonymous authentication is not supported in Grafana Cloud.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/_index.md:102]:
```suggestion
{{% admonition type="note" %}}
Anonymous authentication is not supported in Grafana Cloud.
{{% /admonition %}}

```

File: docs/sources/setup-grafana/configure-security/configure-authentication/_index.md [hunk 4]
```diff
@@ -107,6 +113,8 @@ To disable basic auth:
 enabled = false
 ```
 
+> Please Note: Basic authentication (or Local Grafana Authentication) is not supported in Grafana Cloud.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/_index.md:116]:
```suggestion
{{% admonition type="note" %}}
Basic authentication (or local Grafana authentication) is not supported in Grafana Cloud.
{{% /admonition %}}

```

File: docs/sources/setup-grafana/configure-security/configure-authentication/_index.md [hunk 5]
```diff
@@ -116,6 +124,8 @@ Hide the Grafana login form using the below configuration settings.
 disable_login_form = true
 ```
 
+> Please Note: For Grafana Cloud users, this option can only be configured under specific circumstances, e.g. when enabling LDAP Authentication, please contact Grafana Support for further assistance.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/configure-authentication/_index.md:127]:
```suggestion
{{% admonition type="note" %}}
For Grafana Cloud users, this option can only be configured under specific circumstances, such as enabling LDAP authentication. Please contact Grafana Support for further assistance.
{{% /admonition %}}

```

**Review by chri2547 — APPROVED** [review]
Added some suggestions. Thank you for your contribution!

**Comment by chri2547** [comment]
@SeamusGrafana , you'll want to run prettier. And, do we want to backport to v9.4? Thanks for the contribution!

**Comment by SeamusGrafana** [comment]
> @SeamusGrafana , you'll want to run prettier. And, do we want to backport to v9.4? Thanks for the contribution!

Hi @chri2547 thanks, that should be done now, I think.

Re the Backport, not sure, this is applicable to all of Grafana Cloud across all versions, so not sure if we should backport to all or focus on current and last?

Thanks

**Pull request closed without merging**